inductiva/__init__.py

```diff
@@ -70,17 +73,23 @@ def _check_for_available_package_update():
 
 
 def _check_key():
-    global _checked_key
-
-    if not _checked_key and utils.format_utils.getenv_bool(
-            "GITHUB_ACTIONS", False) is not True:
-        api.methods.validate_api_key(api_key)
-        _checked_key = True
+    if utils.format_utils.getenv_bool("GITHUB_ACTIONS", False) is not True:
+        api.methods.validate_api_key()
```

Comment:
Why remove the verification?

Also, and in this context, `X is not True` is the same as `not X`